src/libraries/Microsoft.VisualBasic.Core/tests/FileSystemTests.cs

```diff
@@ -618,6 +618,8 @@ static void remoteWrite(string fileName, string text)
                             {
                             }
                         }
+
+                        return 0;
```

Comment:
This and the `new RemoteInvokeOptions() { ExpectedExitCode = 0 }` should be deletable, right?